src/app/components/post/view/view.vue

```diff
@@ -86,6 +86,20 @@
 							/>
 						</div>
 
+						<template v-if="shouldShowCommunityPublishError">
+							<br />
+							<div class="well fill-offset">
+								<app-jolticon icon="notice" notice />
+								<span
+									><translate
+										>This post can currently not be published to the selected community channel
```

Comment:
Let's change to:
```
You can't publish this post to the selected community channel because you don't have permissions to post into that specific channel. Please select a different channel.
```